rust/processor/src/processors/mod.rs

```diff
@@ -62,6 +62,14 @@ pub struct ProcessingResult {
     pub db_insertion_duration_in_secs: f64,
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+
+pub struct ProcessedVersions {
+    pub start_version: StartVersion,
+    pub end_version: EndVersion,
+    pub last_transaction_timstamp: Option<aptos_protos::util::timestamp::Timestamp>,
+}
```

Comment:
seems like you can reuse https://github.com/aptos-labs/aptos-indexer-processors/blob/main/rust/processor/src/processors/mod.rs#L58-L64.